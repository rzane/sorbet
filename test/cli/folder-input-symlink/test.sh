#!/bin/bash

main/sorbet --silence-dev-message -p file-table-json --no-stdlib --dir test/cli/folder-input-symlink/input
