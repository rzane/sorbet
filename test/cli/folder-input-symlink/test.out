{
 "files": [
  {
   "path": "test/cli/folder-input-symlink/input/dir-symlink/file.rb",
   "strict": "False",
   "min_error_level": "Max"
  },
  {
   "path": "test/cli/folder-input-symlink/input/file-symlink.rb",
   "strict": "False",
   "min_error_level": "Max"
  },
  {
   "path": "test/cli/folder-input-symlink/input/file.rb",
   "strict": "False",
   "min_error_level": "Max"
  }
 ]
}
